src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -146,13 +146,13 @@
   <ItemGroup Condition=" '$(TargetFramework)' == '$(TargetFrameworkNetAndroid)'">
     <Compile Include="$(PathToMsalSources)\Platforms\Android\**\*.cs" />
     <Compile Include="$(PathToMsalSources)\PlatformsCommon\PlatformNotSupported\ApiConfig\SystemWebViewOptions.cs" />
-    <PackageReference Include="System.Security.SecureString" />
+    <PackageReference Update="System.Security.SecureString" />
```

Comment:
Try adding

`<PackageReference Update="System.Security.SecureString" />`
or if that doesn't work
`<PackageReference Update="System.Security.SecureString" Version="4.3.0"/>`
in the [benchmarks csproj](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/c7610753e4fe72448f88fc614b2a768e6de8f3f7/tests/Microsoft.Identity.Test.Performance/Microsoft.Identity.Test.Performance.csproj#L30).

Reply:
What does "Update" do here? I understand the general logic behind it, as per https://stackoverflow.com/questions/71660693/what-is-the-difference-between-packagereference-update-and-packagereference-incl, but not this.